doc/source/howtos/proxysql-v1.rst

```diff
@@ -0,0 +1,424 @@
+     --sync-multi-cluster-users         Sync user accounts currently configured in MySQL to ProxySQL (Don't delete ProxySQL users not in MySQL)
+     --version, -v                      Print version info
+   
+.. note:: Before using the ``proxysql-admin`` tool,
+   ensure that ProxySQL and |PXC| nodes you want to add are running. For
+   security purposes, please ensure to change the default user settings in
+   the ProxySQL configuration file.
+
+Preparing Configuration File
+----------------------------
+
+It is recommended to provide connection and authentication information
+in the ProxySQL configuration file (:file:`/etc/proxysql-admin.cnf`),
+instead of specifying it on the command line.
+
+By default, the configuration file contains the following:
+
+.. code-block:: text
+
+   # proxysql admin interface credentials.
+   export PROXYSQL_USERNAME="admin"
+   export PROXYSQL_PASSWORD="admin"
+   export PROXYSQL_HOSTNAME="localhost"
+   export PROXYSQL_PORT="6032"
+   
+   # PXC admin credentials for connecting to pxc-cluster-node.
+   export CLUSTER_USERNAME="admin"
+   export CLUSTER_PASSWORD="admin"
+   export CLUSTER_HOSTNAME="localhost"
+   export CLUSTER_PORT="3306"
+   
+   # proxysql monitoring user. proxysql admin script will create this user in pxc to monitor pxc-nodes.
+   export MONITOR_USERNAME="monitor"
+   export MONITOR_PASSWORD="monit0r"
+   
+   # Application user to connect to pxc-node through proxysql
+   export CLUSTER_APP_USERNAME="proxysql_user"
+   export CLUSTER_APP_PASSWORD="passw0rd"
+   
+   # ProxySQL read/write hostgroup 
+   export WRITE_HOSTGROUP_ID="10"
+   export READ_HOSTGROUP_ID="11"
+   
+   # ProxySQL read/write configuration mode.
+   export MODE="singlewrite"
+   
+.. note:: It is recommended to
+   :ref:`change default ProxySQL credentials <default-credentials>`
+   before running ProxySQL in production.
+   Make sure that you provide ProxySQL location and credentials
+   in the configuration file.
+
+Provide superuser credentials for one of the |PXC| nodes.
+The ``proxysql-admin`` script will detect
+other nodes in the cluster automatically.
+
+Enabling ProxySQL
+-----------------
+
+Use the ``--enable`` option to automatically configure a |PXC| node
+into ProxySQL.
+The ``proxysql-admin`` tool will do the following:
+
+* Add |PXC| node into the ProxySQL database
+
+* Add the ``proxysql_galera_checker`` monitoring script
+  into the ProxySQL ``scheduler`` table if it is not available.
+  This script checks for desynced nodes and temporarily deactivates them.
+  It also calls the ``proxysql_node_monitor`` script,
+  which checks cluster node membership
+  and re-configures ProxySQL if the membership changes.
+
+* Create two new |PXC| users with the ``USAGE`` privilege on the node
+  and add them to ProxySQL configuration, if they are not already configured.
+  One user is for monitoring cluster nodes,
```

Comment:
It would be good to mention, that one user is used by ProxySQL to monitor the cluster nodes.  And the other is used by ProxySQL to communicate with the cluster.

Reply:
The following sentence does contain the information you are requesting. Will change them though to mention that these users are used by ProxySQL